sqlitecluster/SQLite.cpp

```diff
@@ -405,7 +409,7 @@ bool SQLite::beginTransaction(bool useCache, const string& transactionName) {
     _sharedData->blockNewTransactionsCV.notify_one();
     SDEBUG("Beginning transaction");
     uint64_t before = STimeNow();
-    _insideTransaction = !SQuery(_db, "starting db transaction", "BEGIN TRANSACTION");
+    _insideTransaction = !timedSQuery("starting db transaction", "BEGIN TRANSACTION");
```

Comment:
I'd skip the timing on these "standard" queries that we run before/after other queries, and on startup/shutdown. I don't think we need to time these. Only "user-supplied" queries should need timing.